packages/web/src/common/store/ui/reactions/slice.ts

```diff
@@ -0,0 +1,83 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { CommonState } from 'common/store'
+import { Nullable } from 'common/utils/typeUtils'
+
+export type ReactionTypes = 'heart' | 'fire' | 'party' | 'explode'
+
+// The order these reactions appear in the web + mobile UI
+export const reactionOrder: ReactionTypes[] = [
+  'heart',
+  'fire',
+  'party',
+  'explode'
+]
+
+export const reactionsMap: { [k in ReactionTypes]: number } = {
```

Comment:
what about sthg like this to make sure the order is always kept?

```
type ReactionTypesMap = { [k in ReactionTypes]: number }
const reactionsMap: ReactionTypesMap = reactionOrder.reduce((acc, curr, i) => ({ ...acc, [curr]: i+1}), {} as ReactionTypesMap)
```

Reply:
Good q - don't think we want to couple the order with the raw value of the reaction here. Imagine we want to change the order on the client someday - we wouldn't want all the old heart reactions to become fire reactions, etc